bundles/org.openhab.ui/web/src/components/pagedesigner/sitemap/attribute-details.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <f7-card v-if="widget">
+    <f7-card-content v-if="attributes.length">
+      <f7-list inline-labels sortable @sortable:sort="onSort">
```

Comment:
Nice idea to allow sorting, to enable sortable mode by default you would need:
```suggestion
      <f7-list inline-labels sortable sortable-opposite class="sortable-enable" @sortable:sort="onSort">
```
The handlers display but sorting doesn't quite work well (the indexes should be recomputed after the sorting and the new order is changed afterwards...)

Reply:
I actually didn't touch this code, but it indeed didn't work. It cost me some effort, but I think I figured out how to make it work. The index as a key doesn't do it, I needed to construct a key that changes when moving an item in the list.

Reply:
It works great 👍